services/121-service/src/activities/interfaces/ignored-duplication.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { ActivityTypeEnum } from '@121-service/src/activities/enum/activity-type.enum';
+import { BaseActivity } from '@121-service/src/activities/interfaces/base-activity.interface';
+
+export interface IgnoredDuplication extends BaseActivity {
+  type: ActivityTypeEnum.IgnoredDuplication;
+  attributes: {
+    duplicateRegistrationId: string;
+    duplicateRegistrationProgramId: string;
+    reason: string | null; // Reason for the status change
```

Comment:
```suggestion
    reason: string | null;
```

I don't think the comment is really necessary.. AND its copied from somewhere, so its also 'incorrect'. :D